docker_push

```diff
@@ -1,5 +1,8 @@
 #!/bin/bash
-echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
-docker build --target production -t aniekano/todo-app
-docker push aniekano/todo-app
+if [[ "$TRAVIS_BRANCH" == "master" ]]; then
```

Comment:
If we did move to the before_deploy step, then we'd no longer need this check since we could rely on travis' `on:` argument to filter out incorrect branches

Reply:
updated